src/patternfly/components/ContextSelector/examples/context-selector-example.hbs

```diff
@@ -82,9 +82,9 @@
   {{#> context-selector-menu}}
     {{#> context-selector-menu-input}}
       {{#> input-group}}
-        {{#> form-control controlType="input" input="true" form-control--attribute='type="search" placeholder="Search" id="textInput2" name="textInput2" aria-label="search through menu items"'}}
+        {{#> form-control controlType="input" input="true" form-control--attribute='type="search" placeholder="Search" id="textInput2" name="textInput2" aria-labelledby="search-button"'}}
         {{/form-control}}
-        {{#> button button--modifier="pf-m-tertiary" button--attribute='aria-label="search button for search input"'}}
+        {{#> button button--modifier="pf-m-tertiary" button--attribute='id="search-button" aria-label="Search menu items"'}}
```

Comment:
id needs to be unique! You should use the same comments that I made above and concat the context-selector--id onto the search button.